### PR TITLE
Fix expansion of RTCP acronym

### PR DESCRIPTION
**Pull request by csperkins**

### CODE REVIEW

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 1]
```diff
@@ -87,7 +87,7 @@
       <t>
         When RTP-based media is used, there are multiple ways to correlate 
         bundled RTP packets with the appropriate "m=" section. This
-        specification defines a new Real-time Transport Control Protocol
+        specification defines a new RTP Control Protocol
```
Comment (cdh4u) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: We should not expand RTP on first occurrence?

Also, the same exists in section 1. Should we do the same change there?

Comment (csperkins) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Yes, agree. I updated the pull request.

Comment (cdh4u) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: So, we should not expand RTP?

Comment (csperkins) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: I thought I did... let me check

Comment (csperkins) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: It's expanded on first use in the abstract.

Comment (cdh4u) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Yes, but normally abbreviations are also expanded on first occurrence outside of the Abstract.

Comment (csperkins) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: I'll revise my pull request expand RTP on first use in the Introduction.

You might want to check the other acronyms though: SDP is expanded on 3rd use in the Intro, for example.

Comment (cdh4u) [reply draft-ietf-mmusic-sdp-bundle-negotiation.xml]: There will be chances to fix the expansions etc. For now we should do the publication request in order to move the document forward :)

But, I will implement your changes and submit a new version of the draft.

**Pull request merged**